CMakeLists.txt

```diff
@@ -15,18 +15,12 @@ add_executable(demo ${DEMO_SOURCE})
 # Dependencies
 add_subdirectory(lib/cubos)
 target_link_libraries(demo cubos-core cubos-engine)
```

Comment:
You also shouldn't need to link `cubos-core`, it is linked by the `cubos-engine` lib.